oml/samplers/distinct_category_balance.py

```diff
@@ -10,24 +10,42 @@
 
 class DistinctCategoryBalanceSampler(IBatchSampler):
     """
-    Let C is a set of categories in dataset, L is a set of labels in dataset:
-    - select n_categories for the 1st batch from C
-    - select n_labels for each of chosen categories for the 1st batch
-    - select n_instances for each label for the 1st batch
-    - define set of available for the 2nd batch labels L*: all the labels from L except the ones
-    chosen for the 1st batch
-    - define set of available categories C*: all the categories corresponding to labels from L*
-    - select n_categories from C* for the 2nd batch
-    - select n_labels for each category from L* for the 2nd batch
-    - select n_instances for each label for the 2nd batch
-    ...
-    If all the categories were chosen sampler resets its state and goes on sampling from the first step.
+    This sampler takes ``n_instances`` for each of the ``n_labels`` for each of the
+    ``n_categories`` to form the batches.
+    Thus, the batch size is ``n_instances x n_labels x n_categories``.
+
+    The strategy for the dataset with ``L`` unique labels and ``C`` unique categories is the following:
+
+    - Select ``n_categories`` of ``C`` for the 1st batch
+
+    - Select ``n_labels`` for each of the chosen categories for the 1st batch
+
+    - Select ``n_instances`` for each of the chosen labels for the 1st batch
+
+    - Define the set of available for the 2nd batch labels ``L*``: these are all the labels ``L`` except the ones chosen for the 1st batch
```

Comment:
check how to add hats or some symbols above instead of `L*`

Reply:
replace to `L^`